iOS/ExposureNotification/source/ApiDefinition.cs

```diff
@@ -15,7 +15,7 @@
 
 namespace ExposureNotifications {
 
-	[Introduced (PlatformName.iOS, 13, 5)]
+	[Introduced (PlatformName.iOS, 12, 5)]
```

Comment:
Just checking if this will cause any issues.

What happens now is that iOS 12.5 has this feature (but not 13.0 - 13.4). It comes back in iOS 13.5.

Reply:
it will not cause runtime issue, the attributes are actually removed by the linker
but it can be confusing to developers, inside the IDE, as it means _since 12.5_